### PR TITLE
Add description for some profiling events

### PR DESCRIPTION
**Pull request by Felixoid**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation (changelog entry is not required)

Detailed description / Documentation draft:

Add missing description to some non obvious profiling events.

### CODE REVIEW

File: dbms/src/Common/ProfileEvents.cpp [hunk 1]
```diff
@@ -20,9 +20,9 @@
     M(ReadBufferAIOReadBytes, "") \
     M(WriteBufferAIOWrite, "") \
     M(WriteBufferAIOWriteBytes, "") \
-    M(ReadCompressedBytes, "") \
-    M(CompressedReadBufferBlocks, "") \
-    M(CompressedReadBufferBytes, "") \
+    M(ReadCompressedBytes, "Number of bytes read from compressed files.") \
```
Comment (alexey-milovidov) [inline dbms/src/Common/ProfileEvents.cpp]: Not necessarily files. Also from network.

Comment (Felixoid) [reply dbms/src/Common/ProfileEvents.cpp]: Is it better now?

**Review by alexey-milovidov — APPROVED** [review]

File: dbms/src/Common/ProfileEvents.cpp (hunk 1 above)
Comment (alexey-milovidov) [reply dbms/src/Common/ProfileEvents.cpp]: Ok.

But the "number of uncompressed bytes read from ..." is not clear enough. The user may be confused and think that some bytes in compressed file are uncompressed and we count them.

Comment (alexey-milovidov) [reply dbms/src/Common/ProfileEvents.cpp]: I've added clarifications.

**Pull request merged**